src/range.rs

```diff
@@ -308,6 +308,100 @@ impl<V: Ord> Range<V> {
     }
 }
 
+/// Implementing `PartialOrd` for start `Bound` of an interval.
+///
+/// ```text
+/// left:   ∞-------]
+/// right:    [-----]
+/// left is smaller, since it starts earlier.
+///
+/// left:   [-----]
+/// right:  ]-----]
```

Comment:
Is the backward brace here a typo? And are these meant to be equal?

Reply:
Oh, maybe it's included/excluded? I'm used to `[` being inclusive and `(` being exclusive.

Reply:
Good point, added a legend